packages/xml-lsp-server/test/server.spec.ts

```diff
@@ -0,0 +1,89 @@
+import * as sinon from "sinon";
+import { assert } from "chai";
+
+import {
+  Settings,
+  configureSettings,
+  validateDocument,
+  SYNTAX_ERROR_MSG
+} from "../src/serverHelper";
+
+import {
+  TextDocument,
+  IConnection,
+  Diagnostic,
+  DiagnosticSeverity,
+  Position
+} from "vscode-languageserver";
+
+describe("Test XML Language Server", () => {
+  let sandbox: any;
+  let doc: TextDocument;
+  let connection: IConnection;
+  let connectionSpy: any;
+
+  before(() => {
+    sandbox = sinon.createSandbox();
```

Comment:
maybe the helper should not be aware of `connection` at all.
and only do logic?